docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-18_remove-analysis-interpolation.txt

```diff
@@ -0,0 +1,8 @@
+* The deprecated module ```iris.analysis.interpolate``` has been removed, along with the following functions:
```

Comment:
Why not add "deprecated" before "functions" as well...

Also, `Linear1DExtrapolator` was a class and not a function, so you should probably turn the last bit of this sentence into "the following deprecated classes and functions:".